nikola/nikola.py

```diff
@@ -2267,9 +2283,12 @@ def atom_link(link_rel, link_type, link_href):
         deps_context["posts"] = [(p.meta[lang]['title'], p.permalink(lang)) for p in
                                  posts]
         deps_context["global"] = self.GLOBAL_CONTEXT
+        deps_context["all_page_dependencies"] = self.ALL_PAGE_DEPENDENCIES
 
         for k in self._GLOBAL_CONTEXT_TRANSLATABLE:
             deps_context[k] = deps_context['global'][k](lang)
+        for k in self._ALL_PAGE_DEPENDENCIES_TRANSLATABLE:
+            deps_dict[k] = deps_dict['all_page_dependencies'][k](lang)
```

Comment:
For globals, that made sense because you wanted to have the "real" (i.e. for current language) value. Here, I'm not sure whether we want only the current value, or the whole dict. Having both doesn't make sense, though, I think.